packages/cms-data-sync/test/utils/assets.test.ts

```diff
@@ -0,0 +1,121 @@
+import { Environment } from 'contentful-management';
+import {
+  checkIfAssetAlreadyExistsInContentful,
+  createAssetLink,
+  migrateAsset,
+} from '../../src/utils';
+import {
+  squidexAsset,
+  contenfulAsset,
+  getContentfulEnvironmentMock,
+  contenfulUploadAssetFields,
+} from '../fixtures';
+
+describe('checkIfAssetAlreadyExistsInContentful', () => {
+  let envMock: Environment;
+  console.log = jest.fn();
```

Comment:
you may want to assign globals back to what they were in an afterAll

Reply:
committed here https://github.com/yldio/asap-hub/pull/2290/commits/5b0a38472172a5267a15b2335793a7516e8c6ca5